python/lsst/dax/imgserv/templates/dali_examples.xml

```diff
@@ -1,13 +1,9 @@
-<!doctype html>
-<title xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html">
-    LSST Image SODA Service</title>
-<link rel=stylesheet type=text/css
-      href="{{ url_for('api_image_soda.static', filename='style.css') }}">
-<div vocab="http://www.ivoa.net/rdf/examples#">
+<?xml version="1.0" encoding="UTF-8"?>
+<div vocab="http://www.ivoa.net/rdf/examples">
 
     <div id="brect" resource="#brect" typeof="example">
-    <p> Example 1: <h2 property="name">Synchronous Query by BRECT </h2>
-        <a href="{{ url_dali_sync }}">
+    <p> Example 1: <h3 property="name">Synchronous Query by BRECT </h3>
+        <a href="{{ url_for('api_image_soda.is_sync', _external=True) }}">
```

Comment:
Ah okay this looks like the flask part that does what you want.  This is good.  Although it would be less confusing if you got rid of the is_.  Does this mean the other thing to determine the capabilities is only called when someone doesn't pass in args?  How are these paths related to each other?

Reply:
The standard calling sequence for user is home portal for imgserv, seeing it supports VO, then /availability, followed by /capabilities, then /examples [optional, not widely supported yet, including sodalint].